options/locale/locale_en-US.ini

```diff
@@ -1989,6 +1990,12 @@ contributors.loading_title_failed = Could not load contributions
 contributors.loading_info = This might take a bit…
 contributors.component_failed_to_load = An unexpected error happened.
 
+recent_commits = Recent Commits
+recent_commits.loading_title = Loading recent commits...
+recent_commits.loading_title_failed = Could not load recent commits
+recent_commits.loading_info = This might take a bit…
+recent_commits.component_failed_to_load = An unexpected error happened.
```

Comment:
Please never ever duplicate translations that already exist! You can rename the ones made for commit graph and use them here. Also, use placeholders if the only thing that changes is the type of data being loaded ("Could not load %s")